NewHorizons/Assets/translations/portuguese_br.json

```diff
@@ -1,6 +1,69 @@
 {
-  "$schema": "https://raw.githubusercontent.com/xen-42/outer-wilds-new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "$schema": "https://raw.githubusercontent.com/outer-wilds-new-horizons/new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "DialogueDictionary": {
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_1": "Sua nave agora está equipada com uma unidade de translocação!",
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_2": "Você pode usar a nova aba \"Modo Interestelar\" no diário de bordo de sua nave para ajustar o piloto automático rumo a outro sistema solar.",
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_3": "Então é só acionar o piloto automático e relaxar!"
+  },
   "UIDictionary": {
-    "Vessel": "Hospedeiro"
+    "INTERSTELLAR_MODE": "Modo Interestelar",
+    "FREQ_STATUE": "Estátua Nomai",
+    "FREQ_WARP_CORE": "Fluxo Anti-Gravitacional",
+    "FREQ_UNKNOWN": "???",
+    "ENGAGE_WARP_PROMPT": "Acionar translocação a {0}",
+    "WARP_LOCKED": "PILOTO AUTOMÁTICO AJUSTADO PARA:\n{0}",
+    "LOCK_AUTOPILOT_WARP": "Ajustar Piloto Automático para sistema solar",
+    "RICH_PRESENCE_EXPLORING": "Explorando {0}.",
+    "RICH_PRESENCE_WARPING": "Translocando para {0}.",
+    "OUTDATED_VERSION_WARNING": "AVISO\n\nO mod New Horizons funciona apenas na versão {0} ou superior. Você está usando a versão {1}.\n\nAtualize o Outer Wilds ou desinstale NH.",
```

Comment:
....
Atualize Outer Wilds ou desinstale NH
(basically without the "o", it ain't needed)

Reply:
agreed, the article is not really necessary here